src/govuk/components/task-list/_index.scss

```
@@ -0,0 +1,120 @@
+@include govuk-exports("govuk/component/task-list") {
+  $govuk-task-list-focus-colour: govuk-colour("black");
+  $govuk-task-list-focus-background-colour: govuk-colour("yellow");
+  $govuk-task-list-hover-colour: govuk-colour("light-grey", $legacy: "grey-3");
+
+  .govuk-task-list {
+    @include govuk-font($size: 19);
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+    list-style-type: none;
+    @include govuk-media-query($from: tablet) {
+      min-width: 550px;
+    }
+  }
+
+  .govuk-task-list__item {
+    margin-bottom: 0;
+    border-bottom: 1px solid $govuk-border-colour;
+  }
+
+  .govuk-task-list__item:first-child {
+    border-top: 1px solid $govuk-border-colour;
+  }
+
+  // This class is added to the <li> elements where the task name is a link.
+  // The background hover colour is added to help indicate that the whole row is clickable, rather
+  // than just the visible link text.
+  .govuk-task-list__item--with-link:hover {
+    background: $govuk-task-list-hover-colour;
+  }
+
+  // This uses table layout so that the task name and status always appear side-by-side, with the width of
+  // each 'column' being flexible depending upon the length of the task names and statuses.
+  .govuk-task-list__item-container {
+    display: table;
+    position: relative;
+    width: 100%;
+    padding-top: govuk-spacing(2);
+    padding-bottom: govuk-spacing(2);
+  }
+
+  .govuk-task-list__task-name-and-hint {
+    display: table-cell;
+    vertical-align: top;
+  }
+
+  .govuk-task-list__status-container {
+    display: table-cell;
+    padding-left: govuk-spacing(2);
+    text-align: right;
+    vertical-align: top;
+  }
+
+  // This adds an empty transparent box covering the whole row, including the task status and
+  // any hint text. Because this is generated within the link element, this allows the whole area
+  // to be clickable.
+  .govuk-task-list__task-link:after {
```

Naming nitpick: as we already have the context that this is part of the task list component and there aren't any other links as part of this component (?) could this be changed to:

```suggestion
  .govuk-task-list__link:after {
```

---

Good idea! Fixed in e9d97ef2e4e6afdbf31c9e4e52ff5f37dd9577d6.